deploy/sundeck_sharing.sql

```diff
@@ -46,10 +46,7 @@ BEGIN
             SNOWFLAKE_SESSION_ID,
             RAW_ACTIONS
         FROM "{SUNDECK_DB}".INTERNAL.GLOBAL_QUERY_HISTORY
-            WHERE UPPER(SNOWFLAKE_ACCOUNT_LOCATOR) = UPPER(CURRENT_ACCOUNT()) and
-            IFF(UPPER(SNOWFLAKE_CLOUD) = 'AZURE', REPLACE(UPPER(SNOWFLAKE_REGION), '-', ''), UPPER(SNOWFLAKE_REGION)) =
-                (SELECT UPPER(region) from "{APPLICATION_PACKAGE}".SHARING.REGIONS WHERE snowflake_region = SPLIT_PART(CURRENT_REGION(), '.', -1)) and
-            UPPER(SNOWFLAKE_CLOUD) = (SELECT UPPER(cloud) from "{APPLICATION_PACKAGE}".SHARING.REGIONS WHERE snowflake_region = SPLIT_PART(CURRENT_REGION(), '.', -1));
+            WHERE  (SELECT UPPER(CURRENT_ACCOUNT()) || region || cloud from "{APPLICATION_PACKAGE}".SHARING.REGIONS WHERE snowflake_region = SPLIT_PART(CURRENT_REGION(), '.', -1)) = PARTITION_KEY;
```

Comment:
Unnecessary for us to be `UPPER(CURRENT_ACCOUNT())`? I did it earlier, trying to be defensive. The value is currently upper-case. Probably unlikely to ever change?

I guess it's less critical that we compute this once, and then compare it against the table data (vs. doing it over every row in the underlying table)

Reply:
fixed